llvm/test/CodeGen/X86/tailcall-cgp-dup.ll

```diff
@@ -362,8 +362,30 @@ return:
   ret ptr %src
 }
 
+@i = global i32 0, align 4
+
+define i32 @undef_tailc() nounwind {
+; CHECK-LABEL: undef_tailc:
+; CHECK:       ## %bb.0:
+; CHECK-NEXT:    cmpl $0, _i(%rip)
+; CHECK-NEXT:    jne _qux ## TAILCALL
+; CHECK-NEXT:  ## %bb.1:
+; CHECK-NEXT:    retq
+  %1 = load i32, ptr @i, align 4
+  %2 = icmp eq i32 %1, 0
+  br i1 %2, label %5, label %3
+
+3:
+  %4 = tail call i32 @qux()
+  br label %5
+
+5:
```

Comment:
Please avoid unnamed values in tests. Especially for block names.

Reply:
Right, thanks for reminding this.